src/test/regress/bin/normalize.sed

```diff
@@ -65,9 +65,6 @@ s/"(raw_events_second_user_id_value_1_key_|agg_events_user_id_value_1_agg_key_)[
 # ignore could not consume warnings
 /WARNING:  could not consume data from worker node/d
 
-# ignore page split with pg13
-/DEBUG:  concurrent ROOT page split/d
```

Comment:
Actually same for this one, it can randomly appear when DEBUG logging is enabled.

Reply:
Good to know, I don't want to increase flakiness in our tests :D

Reply:
probably good to group these two together and improve the comment a bit explaining this for future readers.